### ISSUE TITLE
trouble with 0.5.1

### ISSUE THREAD

**Issue by pdipietro** [jamesmartin/inline_svg#15]
I just bundle update from 0.2.0 to 0.5.1
Now, when starting rails s, it returns the error below, and it's not clear for me how/where to define 

    `asset_finder=': Asset Finder should implement the #find_asset method (InlineSvg::Configuration::Invalid)

Any help or suggestion?
```
pdipietro@CUC-N4J-V4-0-1:~/gsn$ rails s -b0.0.0.0
=> Booting WEBrick
=> Rails 4.2.1 application starting in development on http://0.0.0.0:3000
=> Run `rails server -h` for more startup options
=> Ctrl-C to shutdown server
Exiting
/home/pdipietro/.rvm/gems/ruby-2.1.5/gems/inline_svg-0.5.1/lib/inline_svg.rb:21:in `asset_finder=': Asset Finder should implement the #find_asset method (InlineSvg::Configuration::Invalid)
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/inline_svg-0.5.1/lib/inline_svg/railtie.rb:9:in `block (3 levels) in <class:Railtie>'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/inline_svg-0.5.1/lib/inline_svg.rb:34:in `configure'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/inline_svg-0.5.1/lib/inline_svg/railtie.rb:8:in `block (2 levels) in <class:Railtie>'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:38:in `instance_eval'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:38:in `execute_hook'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:28:in `block in on_load'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:27:in `each'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:27:in `on_load'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/inline_svg-0.5.1/lib/inline_svg/railtie.rb:5:in `block in <class:Railtie>'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/initializable.rb:30:in `instance_exec'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/initializable.rb:30:in `run'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/initializable.rb:55:in `block in run_initializers'
	from /home/pdipietro/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/tsort.rb:226:in `block in tsort_each'
	from /home/pdipietro/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/tsort.rb:348:in `block (2 levels) in each_strongly_connected_component'
	from /home/pdipietro/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/tsort.rb:427:in `each_strongly_connected_component_from'
	from /home/pdipietro/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/tsort.rb:347:in `block in each_strongly_connected_component'
	from /home/pdipietro/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/tsort.rb:345:in `each'
	from /home/pdipietro/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/tsort.rb:345:in `call'
	from /home/pdipietro/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/tsort.rb:345:in `each_strongly_connected_component'
	from /home/pdipietro/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/tsort.rb:224:in `tsort_each'
	from /home/pdipietro/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/tsort.rb:205:in `tsort_each'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/initializable.rb:54:in `run_initializers'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/application.rb:352:in `initialize!'
	from /home/pdipietro/gsn/config/environment.rb:5:in `<top (required)>'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/pdipietro/gsn/config.ru:3:in `block in <main>'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/rack-1.6.0/lib/rack/builder.rb:55:in `instance_eval'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/rack-1.6.0/lib/rack/builder.rb:55:in `initialize'
	from /home/pdipietro/gsn/config.ru:in `new'
	from /home/pdipietro/gsn/config.ru:in `<main>'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/rack-1.6.0/lib/rack/builder.rb:49:in `eval'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/rack-1.6.0/lib/rack/builder.rb:49:in `new_from_string'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/rack-1.6.0/lib/rack/builder.rb:40:in `parse_file'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/rack-1.6.0/lib/rack/server.rb:299:in `build_app_and_options_from_config'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/rack-1.6.0/lib/rack/server.rb:208:in `app'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/commands/server.rb:61:in `app'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/rack-1.6.0/lib/rack/server.rb:336:in `wrapped_app'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/commands/server.rb:139:in `log_to_stdout'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/commands/server.rb:78:in `start'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/commands/commands_tasks.rb:80:in `block in server'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/commands/commands_tasks.rb:75:in `tap'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/commands/commands_tasks.rb:75:in `server'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/railties-4.2.1/lib/rails/commands.rb:17:in `<top (required)>'
	from /home/pdipietro/gsn/bin/rails:8:in `require'
	from /home/pdipietro/gsn/bin/rails:8:in `<top (required)>'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/spring-1.1.3/lib/spring/client/rails.rb:27:in `load'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/spring-1.1.3/lib/spring/client/rails.rb:27:in `call'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/spring-1.1.3/lib/spring/client/command.rb:7:in `call'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/spring-1.1.3/lib/spring/client.rb:26:in `run'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/spring-1.1.3/bin/spring:48:in `<top (required)>'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/spring-1.1.3/lib/spring/binstub.rb:11:in `load'
	from /home/pdipietro/.rvm/gems/ruby-2.1.5/gems/spring-1.1.3/lib/spring/binstub.rb:11:in `<top (required)>'
	from /home/pdipietro/gsn/bin/spring:16:in `require'
	from /home/pdipietro/gsn/bin/spring:16:in `<top (required)>'
	from bin/rails:3:in `load'
	from bin/rails:3:in `<main>'
```

**Comment by jamesmartin** [comment]
@pdipietro, sorry you're having trouble. 

Looks like `inline_svg` believes Sprockets is not loaded in your Rails application. Have you configured the [Asset Pipeline](http://guides.rubyonrails.org/asset_pipeline.html)?

Could you try opening a Rails console and pasting in your `Rails.application.config.assets` configuration, please?

Here's mine, as an example:

```ruby
Loading development environment (Rails 4.2.1)
2.1.1 :001 > Rails.application.config.assets
 => {:_blocks=>[], :paths=>["/Users/jamesmartin/dev/inline_svg_test_app/app/assets/images", "/Users/jamesmartin/dev/inline_svg_test_app/app/assets/javascripts", "/Users/jamesmartin/dev/inline_svg_test_app/app/assets/stylesheets", "/Users/jamesmartin/dev/inline_svg_test_app/vendor/assets/javascripts", "/Users/jamesmartin/dev/inline_svg_test_app/vendor/assets/stylesheets", "/Users/jamesmartin/.rvm/gems/ruby-2.1.1@inline_svg_test_app/gems/turbolinks-2.5.3/lib/assets/javascripts", "/Users/jamesmartin/.rvm/gems/ruby-2.1.1@inline_svg_test_app/gems/jquery-rails-3.1.2/vendor/assets/javascripts", "/Users/jamesmartin/.rvm/gems/ruby-2.1.1@inline_svg_test_app/gems/coffee-rails-4.0.1/lib/assets/javascripts"], :prefix=>"/assets", :manifest=>nil, :precompile=>[#<Proc:0x00000101213cf0@/Users/jamesmartin/.rvm/gems/ruby-2.1.1@inline_svg_test_app/gems/sprockets-rails-2.2.4/lib/sprockets/railtie.rb:60 (lambda)>, /(?:\/|\\|\A)application\.(css|js)$/], :version=>"1.0", :debug=>true, :compile=>true, :digest=>false, :raise_runtime_errors=>true}
```

I'm curious about whether `sprockets` is referenced in your configuration.

**Comment by jamesmartin** [comment]
Looks like the Sprockets interface changed between versions 2 and 3, which I assume you're using as part of your Rails 4.2.1 application. I'll make a fix to accommodate the two different versions and push that out today.

**Comment by jamesmartin** [comment]
@pdipietro I just released [version 0.5.2](https://github.com/jamesmartin/inline_svg/blob/master/CHANGELOG.md#052---2015-04-03), which *should* fix the problem you're seeing.

Please give it a try and let me know if you're still having problems.
